lib/danger/request_source/github.rb

```diff
@@ -231,6 +231,49 @@ def process_markdown(violation)
         message = match.nil? ? html : match.captures.first
         Violation.new(message, violation.sticky)
       end
+
+      # @return [String] The organisation name, is nil if it can't be detected
+      def organisation
+        matched = self.issue_json[:repository_url].match(%r{repos\/(.*)\/})
+        return matched[1] if matched
```

Comment:
you could do `matched && matched[1]` instead of both the return and the `nil` (which is already implicit, since an `if` statement not executed also returns `nil`)